packages/core-web/src/styles/index.css

```diff
@@ -1,3 +1,5 @@
 @import "markbind.css";
 @import "page-nav.css";
 @import "site-nav.css";
+@import url("~katex/dist/katex.css");
```

Comment:
what's the additional `url` for?

Reply:
![Untitled](https://user-images.githubusercontent.com/3306138/91737768-1a9f3880-ebe2-11ea-91ba-69631652368a.png)

You'll need to copy the font files to the appropriate output directory too

Reply:
The additional `url` was how I used to import other css files. Upon checking https://developer.mozilla.org/en-US/docs/Web/CSS/@import, it seems that `url` is redundant. I'll remove it.

Reply:
![image](https://user-images.githubusercontent.com/38814428/92304834-46406b00-efb4-11ea-90a1-84a8c3e9a43b.png)
I've fixed the issue, added extra logic in Site to copy over the files to the appropriate directory! (imo, that was more straightforward compared to two different webpack configurations for dev and prod)